setup.py

```diff
@@ -28,7 +28,9 @@
     "generic-parser>=1.0.6",
     "sdds>=0.1.3",
     "h5py>=2.9.0",
+    "tables>=3.6.0",
```

Comment:
The `pandas` doc [here](https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html) say the minimum requirement on `PyTables` is `3.5.1`, maybe we use that version instead to ease things? Is there a reason we need `>=3.6.0`?

Reply:
3.5.1 has no wheels for windows or macos >py3.7